tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -79,43 +112,66 @@ def test_vm_snapshot_ops(
         4. Create new vm using restored pvc Verify existing data of the VM are not changed.
         5. Add further data(e.g., new file) to the VM
         6. Repeat the above procedure for all the VMs in the system
-        7. Delete all the VMs created as part of this test
+        7. Stop all the VMs created as part of this test.
         """
+        proj_obj = project_factory()
         file_paths = ["/file.txt", "/new_file.txt"]
-        # TODO: Add multi_cnv fixture to configure VMs based on specifications
-        vm_obj = cnv_workload(
-            volume_interface=constants.VM_VOLUME_PVC,
-            source_url=constants.CNV_FEDORA_SOURCE,
-        )[0]
-        # Writing IO on source VM
-        source_csum = run_dd_io(vm_obj=vm_obj, file_path=file_paths[0], verify=True)
-        # Stopping VM before taking snapshot of the VM PVC
-        vm_obj.stop()
-        # Taking Snapshot of PVC
-        pvc_obj = vm_obj.get_vm_pvc_obj()
-        snap_obj = snapshot_factory(pvc_obj)
-        # Restore the snapshot
-        res_snap_obj = snapshot_restore_factory(
-            snapshot_obj=snap_obj,
-            storageclass=vm_obj.sc_name,
-            size=vm_obj.pvc_size,
-            volume_mode=snap_obj.parent_volume_mode,
-            access_mode=vm_obj.pvc_access_mode,
-            status=constants.STATUS_BOUND,
-            timeout=300,
+        vm_objs_def, vm_objs_aggr, _, _ = multi_cnv_workload(
```

Comment:
Team decision: Let's create the VMs in parallel to reduce the execution time. This may be done as part a separate PR